File: RVAS/UserControls/Bootstrap/Bootstrap_Callout.ascx.vb
Public Class Bootstrap_Callout
    Inherits System.Web.UI.UserControl

    Protected Sub Page_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load

    End Sub
    Public Property Display() As Boolean

        Get
            If divCallout.Style("display") = "" Then
                Return True
            Else
                Return False
            End If
        End Get

        Set(ByVal value As Boolean)
            If value = True Then
                divCallout.Style("display") = ""
            Else
                divCallout.Style("display") = "none"
            End If
        End Set
    End Property

    Public Property Message() As String
        Get
            Return lblMessage.Text
        End Get
        Set(ByVal value As String)
            lblMessage.Text = value
        End Set
    End Property
    Public Property ShowWarning() As Boolean
        Get
            If iconClass.Attributes("class") = "icon fa fa-ban" Then
                Return True
            Else
                Return False
            End If

        End Get
        Set(ByVal value As Boolean)

            If value = True Then

                iconClass.Attributes("class") = "icon fa fa-ban"
                divCallout.Attributes("class") = "callout callout-danger"
                lblTitle.Text = "خطا"

            Else

                iconClass.Attributes("class") = "icon fa fa-info"
                divCallout.Attributes("class") = "callout callout-info"
                lblTitle.Text = "توجه"
            End If
        End Set
    End Property

End Class

File: SmartInterface/UserControls/Bootstrap/Bootstrap_Callout.ascx.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

namespace SmartInterface.UserControls.Bootstrap
{
    public partial class Bootstrap_Callout : System.Web.UI.UserControl
    {
        protected void Page_Load(object sender, EventArgs e)
        {

        }

        public bool Display {
            get
            {
                if (divCallout.Style["display"] == "")
                    return true;
                else
                    return false;
            }
            set
            {
                if (value)
                    divCallout.Style["display"] = "";
                else
                    divCallout.Style["display"] = "none";
            }

        }

        public string Message {
            get {
                return lblMessage.Text;
            }
            set {
                lblMessage.Text = value;
            }
         }

        public bool ShowWarning {
            get
            {
                if (iconClass.Attributes["class"] == "icon fa fa-ban")
                    return true;
                else
                    return false;
            }
            set
            {
                if (value)
                {
                    iconClass.Attributes["class"] = "icon fa fa-ban";
                    divCallout.Attributes["class"] = "callout callout-danger";
                    lblTitle.Text = "خطا";
                }
                else
                {
                    iconClass.Attributes["class"] = "icon fa fa-info";
                    divCallout.Attributes["class"] = "callout callout-info";
                    lblTitle.Text = "توجه";
                }
            }
        }

    }
}